composer.json

```diff
@@ -115,7 +115,7 @@
             "@test:integration"
         ],
         "test:unit": "phpunit -c tests/phpunit.unit.xml",
-        "test:integration": "phpunit -c tests/phpunit.integration.xml",
+        "test:integration": "phpunit -c tests/phpunit.integration.xml --process-isolation",
```

Comment:
let's not forget to remove this from here and rebase to the master branch

Reply:
Not sure how I missed this one :see_no_evil: Thanks!